### DISCUSSION TITLE
Remotepayload for wix for .net 4.8 nned to update for every new release?

### DISCUSSION THREAD

**Discussion by gauravmlmishra** [wixtoolset/issues#6528] [Questions] [answered]
how can correctly use RemotePayload so that i do not need to modify it for every new release.
I have referred the wxs file https://github.com/wixtoolset/wix3/blob/develop/src/ext/NetFxExtension/wixlib/NetFx48.wxs
but it doesn't work any more because now the download link 
 https://go.microsoft.com/fwlink/?LinkId=2088631
points to the newer version of .net 4.8 .If this link would have been permanent then this solution would have worked.
As of today this links points to 4.8.4115.0 and not 4.8.3928.0 .I have generated new CertificatePublicKey, hash etc for the new version.
But what if next week the link again starts pointing to a newer version. Then my setup in the filed will stop working.
What is the way out ? Is my understanding of how this works correct?

**Comment by robmen** [comment, marked as answer]
In WiX v3, if you use the certificate verification, this problem only arises when Microsoft changes their signing certificate. Something they did recently after not doing so for... many, many years. So, your understanding is correct.
